src/helpers/strings-es.ts

```diff
@@ -317,7 +317,50 @@
     "device-data-month-chart-no-data": "Sin Datos",
     "device-data-month-chart-daily-average": "Promedio Diario",
     "device-data-month-chart-minutes": "Minutos",
-    "device-data-month-chart-sleep": "Dormir"
```

Comment:
Any idea why the line count is off here?  Are we missing a few Spanish strings higher up in the file?  The rest of the language files all seem to be in sync on the line numbers.

Reply:
It's consistent now after I merged in main, probably a missing string I copied in from main or something